kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/sli_metrics.py

```diff
@@ -0,0 +1,87 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import division
+
+from copy import deepcopy
+
+from datadog_checks.base.checks.openmetrics import OpenMetricsBaseCheck
+
+SLI_METRICS_PATH = '/slis'
+
+SLI_METRICS_MAP = {
+    'kubernetes_healthcheck': 'kubernetes_healthcheck',
+    'kubernetes_healthchecks_total': 'kubernetes_healthchecks_total',
+}
+
+
+class SliMetricsScraperMixin(OpenMetricsBaseCheck):
```

Comment:
As this scraper mixin is new, I’m wondering if it shouldn’t use [the V2 of the OpenMetrics base check](https://github.com/DataDog/integrations-core/blob/master/docs/developer/base/openmetrics.md) since the beginning.

```suggestion
from datadog_checks.base.checks.openmetrics.v2.scraper import OpenMetricsScraper

SLI_METRICS_PATH = '/slis'

SLI_METRICS_MAP = {
    'kubernetes_healthcheck': 'kubernetes_healthcheck',
    'kubernetes_healthchecks_total': 'kubernetes_healthchecks_total',
}


class SliMetricsScraperMixin(OpenMetricsScraper):
```

@DataDog/agent-integrations can confirm, but I think that `OpenMetricsBaseCheck` should be deprecated.

Reply:
As migrating the `kube_apiserver` as well to `OpenMetricsBaseCheckV2` seems to be less straightforward, planning to keep this mixin on V1 to stay consistent

Reply:
@jennchenn just qq: does this functionality exist in v2 already? If not, what's the rationale to add it to v1 first?

@L3n41c is correct that we're no longer actively developing v1 of the OM base check, that's why I ask.

Reply:
@jennchenn thanks for the context. I have updated the PR description to mention why we're not adding these metrics to V2. I will resolve this conversation and re-review the PR after we deal with the other comments I left on it.